src/Docs.scss

```diff
@@ -0,0 +1,13 @@
+.docs {
+  .sidebar {
+    min-width: 100px;
+    max-width: 200px;
+    @media screen and (max-width: 800px) {
+      display: none;
+    }
+    .header {
+      margin: 10px auto 0 auto;
+      text-align: center;
```

Comment:
Are the .header ones necessary? I'm trying to minimize subjective changes to css.